# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/rust/tabs/.cargo-checksum.json
# Commit: 2147c5c81edf
# Full Hash: 2147c5c81edf9cd56c19e7e92dfebfcd77329978
# Author: Ryan VanderMeulen <ryanvm@gmail.com>
# Date: 2023-03-01 17:52 -0500
# Regressor Bug: 1818349
# File Overlap Count: 24
# Overlapping Files: third_party/rust/sync15/src/client/coll_update.rs, third_party/rust/sync15/src/client/mod.rs, third_party/rust/sync15/.cargo-checksum.json, third_party/rust/sync15/src/client/sync.rs, Cargo.toml
# Description:
#   Backed out changesets b05343a5b533 and 57b2cda7e0db (bug 1818349) for causing bug 1819723.
# ==============================================================================

diff -r d59b76766f0d -r 2147c5c81edf third_party/rust/tabs/.cargo-checksum.json
--- a/third_party/rust/tabs/.cargo-checksum.json	Wed Mar 01 05:53:30 2023 +0000
+++ b/third_party/rust/tabs/.cargo-checksum.json	Wed Mar 01 17:52:01 2023 -0500
@@ -1,1 +1,1 @@
-{"files":{"Cargo.toml":"8ae43dd6969feea0eabc5c53d830960212e6cb1a18b43a00f47e8784c2fd0063","README.md":"c48b8f391ef822c4f3971b5f453a1e7b43bea232752d520460d2f04803aead1a","build.rs":"33e61b811b19ed2b58e319cc65d5988bed258d2c4fea2d706301184c59847a0f","src/error.rs":"ac3d450f0ba6a855c37fa2dd829004b22dce5ad4416ebec66a3d7d6212bdbcd7","src/lib.rs":"7208f78955e015ef8bab7916307e551cd3c1bd56d7fe14f8b53cd53bc4b38555","src/schema.rs":"2b7b51f3c2edc0ca603495c10b917603fd9ac791c4a366080e40d090b13b91f2","src/storage.rs":"eb9505477a926919769b09988ebbeeba96ceada634aec79b73fa75f79c038b03","src/store.rs":"ab0b6214b30b0f0fa7c6a89098ff3db1a8f76264f6711c4481c0be460afe522b","src/sync/bridge.rs":"705a24934c90494c2074288d03266504a866c4f04cb89c4e17b762cb498d621e","src/sync/engine.rs":"f2c5e4a2cb400ca33b649a078fc56e59a01a81a6e16e42b126e8b00d22b5eeea","src/sync/full_sync.rs":"47660644b3a66f3adff1130bb64e2ef40cb29749a40bc28d71246641d28ad080","src/sync/mod.rs":"81f98303b2a60aa6c6ed0d9777a95d6bbff46e57a494a0172d81c5819296802e","src/sync/record.rs":"896ebc6aa213bac9e6170c7e0b7dbee322f62e5f6c28462cb6da0bfe8ce938ba","src/tabs.udl":"a555fe11b5fa7ea9aefa7d7be31906a63b31cbc16b9b7f5ad952fd0e08ba5c61","uniffi.toml":"5156701368f0b5856e658143714a43058385c8ac53bee72d7a5a332b576dfb82"},"package":null}
\ No newline at end of file
+{"files":{"Cargo.toml":"8ae43dd6969feea0eabc5c53d830960212e6cb1a18b43a00f47e8784c2fd0063","README.md":"c48b8f391ef822c4f3971b5f453a1e7b43bea232752d520460d2f04803aead1a","build.rs":"33e61b811b19ed2b58e319cc65d5988bed258d2c4fea2d706301184c59847a0f","src/error.rs":"ac3d450f0ba6a855c37fa2dd829004b22dce5ad4416ebec66a3d7d6212bdbcd7","src/lib.rs":"7208f78955e015ef8bab7916307e551cd3c1bd56d7fe14f8b53cd53bc4b38555","src/schema.rs":"2b7b51f3c2edc0ca603495c10b917603fd9ac791c4a366080e40d090b13b91f2","src/storage.rs":"0a74e341d33b6734f6ef0130e2ba506ec313b5a5f8d6660c535ffc53520edabc","src/store.rs":"ab0b6214b30b0f0fa7c6a89098ff3db1a8f76264f6711c4481c0be460afe522b","src/sync/bridge.rs":"a444c97991cfa8e0dd92ba3b35c6864ddd0cfec1d17e5cc514b9c36ada8d46c3","src/sync/engine.rs":"bbee416c9d45b90b945016d26c2133639c1ad057c6f52fdbdfa5a4f4f631a378","src/sync/full_sync.rs":"514668bb76d1fe6de3a36e64e6984b8dbd391155685ea21cbf91f67a957babdd","src/sync/mod.rs":"81f98303b2a60aa6c6ed0d9777a95d6bbff46e57a494a0172d81c5819296802e","src/sync/record.rs":"896ebc6aa213bac9e6170c7e0b7dbee322f62e5f6c28462cb6da0bfe8ce938ba","src/tabs.udl":"a555fe11b5fa7ea9aefa7d7be31906a63b31cbc16b9b7f5ad952fd0e08ba5c61","uniffi.toml":"5156701368f0b5856e658143714a43058385c8ac53bee72d7a5a332b576dfb82"},"package":null}
\ No newline at end of file